sklearn/preprocessing/tests/test_target_encoder.py

```diff
@@ -125,7 +129,7 @@ def test_encoding(categories, unknown_value, global_random_seed, smooth, target_
     assert_allclose(target_encoder.encodings_[0], expected_encodings)
     assert target_encoder.target_mean_ == pytest.approx(y_mean)
 
-    # Transform on test data, the last value is unknown is it is encoded as the target
+    # Transform on test data, the last value is unknown so it is encoded as the target
```

Comment:
Can you open another PR for the grammar corrections? (To make it easier for the second reviewer)